doc/cephfs/fs-volumes.rst

```diff
@@ -97,6 +97,43 @@ The CephX IDs authorized to <vol_name> need to be reauthorized to <new_vol_name>
 on-going operations of the clients using these IDs may be disrupted. Mirroring is
 expected to be disabled on the volume.
 
+Fetch the information of a CephFS volume using::
+
+    $ ceph fs volume info vol_name
+    {
+        "mon_addrs": [
+            "192.168.1.7:40280"
+        ],
+        "pending_subvolume_deletions": 0,
+        "pools": {
+            "data": [
+                {
+                    "avail": "106.288848896 GiB",
+                    "name": "cephfs.vol_name.data",
+                    "used": "0.0 GiB"
+                }
+            ],
+            "metadata": [
+                {
+                    "avail": "106.288848896 GiB",
+                    "name": "cephfs.vol_name.meta",
+                    "used": "0.000135168 GiB"
+                }
+            ]
+        },
+        "size": "0 B"
+    }
+
+The output format is json and contains fields as follows.
+
+* pools: Data and metadata pools related information
+        * avail: The amount of free space available in GiB
```

Comment:
The first line is a bit stilted, suggest: "*pools: Attributes of data and metadata pools"